lib/orderbook/types.ts

```diff
@@ -1,6 +1,12 @@
 import { SwapClientType } from '../constants/enums';
 import { SwapSuccess, SwapFailure } from '../swaps/types';
 
+export type OrderBookThresholds = {
+  disable: boolean;
```

Comment:
Rather than a separate `disable` property, how about we just treat a `minQuantity` of 0 as the threshold being disabled?